cli_tools/gce_windows_upgrade/main.go

```diff
@@ -0,0 +1,75 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// gce_windows_upgrade is a tool for upgrading GCE Windows instances.
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging/service"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image_export/exporter"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_windows_upgrade/upgrader"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+// project/zone/network/subnet/labels are not required when the upgrade has to happen in place.
+var (
```

Comment:
How will future upgrade paths (e.g. 2012->2016) be supported? Is there a flag needed or will it be auto detected?